src/saltext/proxmox/clouds/proxmox.py

```diff
@@ -1096,7 +1076,7 @@ def show_instance(name, call=None):
     return nodes[name]
 
 
-def get_vmconfig(vmid, node=None, node_type="openvz"):
+def get_vmconfig(vmid, node=None, node_type="lxc"):
```

Comment:
**question (blocking):** is there any reasoning behind the default value here? is other words, is there anything inherent about the proxmox code that favors one as a default or a sense of what the most "popular" deployments are?